.github/workflows/quality.yaml

```diff
@@ -1,5 +1,8 @@
 name: Quality
 
+permissions:
```

Comment:
hmmm this is not needed on lighteval: https://github.com/huggingface/lighteval/blob/main/.github/workflows/quality.yaml